manifest/sso-int/manifest.json

```diff
@@ -60,6 +60,6 @@
   ],
   "webApplicationInfo": {
     "id": "ab93102c-869b-4d34-a921-a31d3e7f76ef",
-    "resource": "api://taskmeow.ngrok.io/ab93102c-869b-4d34-a921-a31d3e7f76ef"
+    "resource": "api://taskmeow.ngrok.io/botid-ab93102c-869b-4d34-a921-a31d3e7f76ef"
```

Comment:
What is the purpose of the `botid-` prefix here?

Reply:
we need to update the resource URI in of the AAD application to include the botid. Teams client is going to check that the bot requesting the token is the same as the one in the resource.